src/libs/actions/Report.js

```diff
@@ -1638,23 +1638,31 @@ function shouldShowReportActionNotification(reportID, action = null, isRemote =
 
 /**
  * @param {String} reportID
- * @param {Object} action
+ * @param {Object} reportAction
  */
-function showReportActionNotification(reportID, action) {
-    if (!shouldShowReportActionNotification(reportID, action)) {
+function showReportActionNotification(reportID, reportAction) {
+    if (!shouldShowReportActionNotification(reportID, reportAction)) {
         return;
     }
 
     Log.info('[LocalNotification] Creating notification');
-    LocalNotification.showCommentNotification({
-        report: allReports[reportID],
-        reportAction: action,
-        onClick: () => {
-            // Navigate to this report onClick
-            Navigation.navigate(ROUTES.REPORT_WITH_ID.getRoute(reportID));
-        },
-    });
-    notifyNewAction(reportID, action.actorAccountID, action.reportActionID);
+    const report = allReports[reportID];
+
+    if (reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.MODIFIEDEXPENSE) {
+        LocalNotification.showModifiedExpenseNotification({
+            report,
+            reportAction,
+            onClick: () => Navigation.navigate(ROUTES.getReportRoute(reportID)),
+        });
+    } else {
+        LocalNotification.showCommentNotification({
+            report,
+            reportAction,
+            onClick: () => Navigation.navigate(ROUTES.getReportRoute(reportID)),
+        });
```

Comment:
NAB these params looked the same and could be DRY'd up

```suggestion
    const notificationParams = {
        report,
        reportAction,
        onClick: () => Navigation.navigate(ROUTES.getReportRoute(reportID)),
    };
    if (reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.MODIFIEDEXPENSE) {
        LocalNotification.showModifiedExpenseNotification(notificationParams);
    } else {
        LocalNotification.showCommentNotification(notificationParams);
```